glide.lock

```diff
@@ -1,215 +1,16 @@
-hash: 9589cf38139f18ca918955308856f05f0f5196ca6189cd6961ffd1aff19d8ba6
```

Comment:
Is it that the deps are installed as git submodules ? or should they be added exploded (duplicated) ? Hmm ...